Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.yml

```diff
@@ -1939,6 +1945,27 @@ script:
     - contextPath: MicrosoftATP.FileStatistics.Statistics.TopFileNames
       description: The file's top names.
       type: String
+    - contextPath: File.SHA1
+      description: The SHA1 hash of the file.
+      type: String
+    - contextPath: File.OrganizationPrevalence
+      description: How frequently the indicator is detected within a particular organization.
```

Comment:
match according to Dror's answer above & consult with tech-writing

Reply:
Frequency is "once per week" or "twice per day". The number of times is how many times  it's been detected since the beginning.
Which is the correct description?
I think it should be the number of times.

Reply:
Fixed based on your suggestions to use the phrase "The number of times" instead of "How frequently"